newIDE/app/src/Utils/GDevelopServices/Asset.js

```diff
@@ -208,10 +208,20 @@ export const listAllPublicAssets = async ({
     throw new Error('Unexpected response from the assets endpoints.');
   }
 
+  const publicAssetShortHeaders = responsesData[0];
+  const publicFilters = responsesData[1];
+  const publicAssetPacks = responsesData[2];
+
+  if (!publicAssetPacks.starterPacks) {
```

Comment:
for some reason, sometimes the response is not null but doesn't contain starterPacks. Preventing this

Reply:
Sounds good